internal/command/untaint.go

```diff
@@ -163,14 +165,24 @@ func (c *UntaintCommand) Run(args []string) int {
 		c.showDiagnostics(diags)
 		return 1
 	}
+
+	// Get schemas, if possible, before writing state
+	var schemas *terraform.Schemas
+	if isCloudMode(b) {
```

Comment:
It feels kinda unfortunate that this idea of "cloud mode" is infecting so many codepaths now. I don't really have a better idea to suggest, though. :confounded: Hopefully we can think about a more elaborate refactoring of this later, but this surgical approach is probably best for now.